src/vm/hints/secp/ec_utils.rs

```diff
@@ -247,4 +400,335 @@ mod tests {
             )))
         );
     }
+
+    #[test]
+    fn run_compute_slope_ok() {
+        let hint_code = "from starkware.cairo.common.cairo_secp.secp_utils import SECP_P, pack\nfrom starkware.python.math_utils import line_slope\n\n# Compute the slope.\nx0 = pack(ids.point0.x, PRIME)\ny0 = pack(ids.point0.y, PRIME)\nx1 = pack(ids.point1.x, PRIME)\ny1 = pack(ids.point1.y, PRIME)\nvalue = slope = line_slope(point1=(x0, y0), point2=(x1, y1), p=SECP_P)";
+        let mut vm = VirtualMachine::new(
+            VM_PRIME.clone(),
+            vec![(
+                "range_check".to_string(),
+                Box::new(RangeCheckBuiltinRunner::new(true, bigint!(8), 8)),
+            )],
+            false,
+            &HINT_EXECUTOR,
+        );
```

Comment:
There's a newly added macro for this, `vm_with_range_check!` IIRC.

Reply:
Nice, didnt see it.
Done